ci-operator/step-registry/openshift/microshift/infra/sos-aws/openshift-microshift-infra-sos-aws-commands.sh

```diff
@@ -7,6 +7,142 @@ source /tmp/ci-functions.sh
 ci_script_prologue
 trap_subprocesses_on_term
 
+trap 'generate_junit' EXIT TERM INT
```

Comment:
This overrides the subprocess TERM trap, so we need something that runs both, probably?